src/components/formProducts/FormProducts.js

```diff
@@ -99,19 +97,40 @@ const FormProducts = ({ items }) => {
         </div>
         <div className="productos_view products-container">
           {itemsFiltered?.length > 0 ? (
-            itemsFiltered.map((item, index) => (
-              <ProductForList
-                key={`${index}-${item.name}`}
-                item={item}
-                token={token.current}
+            <>
+              <ProductsByEstimation
+                items={itemsFiltered}
+                start={0}
+                end={7}
+                title="Soon"
+                estimationType="soon"
               />
-            ))
+              <ProductsByEstimation
+                items={itemsFiltered}
+                start={8}
+                end={15}
+                title="Kind of Soon"
+                estimationType="kind-of-soon"
+              />
+              <ProductsByEstimation
+                items={itemsFiltered}
+                start={16}
+                end={31}
```

Comment:
Valores de las propiedades `start` and `end` **no coincide**n con el AC definido en el [issue 12](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/issues/12).

>  No necesita comprarlo pronto (más de 30 días)

Por favor, te agradecería su cambio:

```suggestion
                end={31}
```